cmd/cosign/cli/options/certificate.go

```diff
@@ -43,6 +44,9 @@ func (o *CertVerifyOptions) AddFlags(cmd *cobra.Command) {
 	cmd.Flags().StringVar(&o.CertEmail, "certificate-email", "",
 		"the email expected in a valid Fulcio certificate")
 
+	cmd.Flags().StringVar(&o.CertIdentity, "certificate-identity", "",
+		"the identity expected in a valid Fulcio certificate")
```

Comment:
I'd like to see a little bit more detail in this help text. What's an "identity" vs. an "email"?

Reply:
I added a list of acceptable types of values. WDYT?

Reply:
Looks great!